README.md

```diff
@@ -4,21 +4,108 @@
+You can download latest master builds from [Bintray](https://dl.bintray.com/ocdev/ocdev/latest/) or 
+builds for released versions from [GitHub releases page](https://github.com/kadel/ocdev/releases).
+
+### MacOS
+1. First you need enable `kadel/ocdev` Homebrew Tap:
+    ```sh
+    brew tap kadel/ocdev
+    ```
+2. 
+    - If you want to install latest master build
+    ```sh
+    brew install kadel/ocdev/ocdev -- HEAD
+    ```
+    - If you want to install latest released version
+    ```sh
+    brew install kadel/ocdev/ocdev
+    ```
+
+### Linux
+#### Debian/Ubuntu and other distributions using deb
+1. First you need to add gpg [public key](https://bintray.com/user/downloadSubjectPublicKey?username=bintray) used to sign repositories.
+    ```sh
+    curl https://bintray.com/user/downloadSubjectPublicKey?username=bintray | apt-key add -
+    ```
+2. Add ocdev repository to your `/etc/apt/sources.list`
+    - If you want to use latest master builds add  `deb https://dl.bintray.com/ocdev/ocdev-deb-dev strech main` repository.
+      ```sh
+      echo "deb https://dl.bintray.com/ocdev/ocdev-deb-releases strech main" | sudo tee -a /etc/apt/sources.list
+      ```
+    - If you want to use latest released version add  `deb https://dl.bintray.com/ocdev/ocdev-deb-releases strech main` repository.
+      ```sh
+      echo "deb https://dl.bintray.com/ocdev/ocdev-deb-dev strech main" | sudo tee -a /etc/apt/sources.list
+      ```
+3. Now you can install `ocdev` and you would install any other package.
+   ```sh
+   yum install ocdev
+   # or 'dnf install ocdev'
+   ```
+
+
+#### Fedora/Centos/RHEL and other distribution using rpm
+1. Add ocdev repository to your `/etc/yum.repos.d/`
+    - If you want to use latest master builds save following text to `/etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo`
+        ```
+        # /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo
+        [bintraybintray-ocdev-ocdev-rpm-dev]
+        name=bintray-ocdev-ocdev-rpm-dev
+        baseurl=https://dl.bintray.com/ocdev/ocdev-rpm-dev
+        gpgcheck=0
+        repo_gpgcheck=0
+        enabled=1
+        ```
+        Or you can download it using following command:
+        ```sh
+        sudo wget https://bintray.com/ocdev/ocdev-rpm-dev/rpm -O /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo
+        ```
+    - If you want to use latest released version save following text to `/etc/yum.repos.d/bintray-ocdev-ocdev-rpm-releases.repo`
+        ```
+        # /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-releases.repo
+        [bintraybintray-ocdev-ocdev-rpm-releases]
+        name=bintray-ocdev-ocdev-rpm-releases
+        baseurl=https://dl.bintray.com/ocdev/ocdev-rpm-releases
+        gpgcheck=0
+        repo_gpgcheck=0
+        enabled=1
+        ```
+        Or you can download it using following command:
+        ```sh
+        sudo wget https://bintray.com/ocdev/ocdev-rpm-releases/rpm -O /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-releases.repo
+        ```
```

Comment:
Wondering why you do not use https://copr.fedorainfracloud.org/ for this. This way it is the standard what way for RPMs those are not included in the distribution. You can build it for Fedora (multiple versions) and EL.

Reply:
Because, to be honest creating proper rpm SPEC for go project with vendor sucks :-( and it is hard to automate (especially with covering vendor updates). It would take me multiple days to do it. This was just couple of hours of work and it is working ;-)

I would rather have at least some package than no package at all.

Reply:
Fair enough. We can start with what you have.